templates/app/index.html

```diff
@@ -6,7 +6,8 @@
     <title><%= name %></title>
     <meta name="description" content="<%= name %> : <%= description %>">
     <link rel="stylesheet" type="text/css" media="screen" href="styles/stylesheet.css">
-    <script src="scripts/offline-manager.js"></script>
+    <script data-worker="offliner-worker.js" src="lib/offliner/dist/offliner-client.min.js"></script>
+    <script defer src="scripts/index.js"></script>
```

Comment:
The structure of the &lt;script> tag that enables offlining is a key part of the interface design of Oghliner, especially since its addition to the page(s)/template(s) of a pre-existing app is one of the only parts of the *integrate* process that is not fully automated. So it's important that it be as simple and straightforward as possible.

Thus there are a few issues here:

1. Renaming the "manager" script to index.js is dangerous, because that name could conflict with another script in a pre-existing app. The original name, or another name that is less likely to conflict, would be better. (Of course this isn't a problem in the template app, but it'd be simpler to keep the name identical regardless of whether the file is being generated by *bootstrap* or *integrate*.).

2. Renaming the worker script to offliner-worker.js is ok, but the reason I called it offline-worker.js instead of oghliner-worker.js or sw-precache-worker.js is that I didn't want to expose the implementation detail of which node module is being used to generate the code, since that detail is irrelevant to a developer who is looking at a script reference and wondering what the script does.

3. Requiring a developer to add and maintain two script tags instead of one is double the effort, and it's additional effort when one of them has a complicated *src* path and an additional, nonstandard *data-worker* attribute. Is it possible to integrate the functionality provided by offliner-client.min.js into the "manager" script, such the developer only has to integrate and maintain a single, simple script reference? It's fine to call it a "client" rather than a "manager", if that's more accurate; the key is just to simplify the code to reduce the developer burden to the absolute minimum. The ideal interface here is: 

```html
<script src="offline-client.js"></script>
```
(In practice, we end up putting the file into a "scripts/" subdirectory, which is more conventional for web software projects.)